livedata/src/test/java/io/getstream/chat/android/livedata/controller/helper/MessageHelperTests.kt

```diff
@@ -0,0 +1,113 @@
+package io.getstream.chat.android.livedata.controller.helper
+
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.mock
+import io.getstream.chat.android.client.helpers.AttachmentHelper
+import io.getstream.chat.android.livedata.randomAttachment
+import io.getstream.chat.android.livedata.randomMessage
+import org.amshove.kluent.When
+import org.amshove.kluent.calling
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.util.Date
+
+internal class MessageHelperTests {
+
+    private lateinit var sut: MessageHelper
+    private lateinit var attachmentHelper: AttachmentHelper
+
+    @BeforeEach
+    fun setup() {
+        attachmentHelper = mock()
+        sut = MessageHelper(attachmentHelper)
+    }
+
+    @Test
+    fun `Should return list with the same size as new messages with same messages`() {
+        val message1 = randomMessage()
+        val message2 = randomMessage()
+
+        val result = sut.updateValidAttachmentsUrl(listOf(message1, message2), emptyMap())
+
+        result.size shouldBeEqualTo 2
+        result.any { it === message1 } shouldBeEqualTo true
+        result.any { it === message2 } shouldBeEqualTo true
```

Comment:
Why don't we check the result list is equals to `listOf(message1, message2)`?

Reply:
You're right. It's more obvious)